packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -59,6 +59,7 @@ const SettingsSidebar = () => {
 
 	return (
 		<PluginSidebarEditPost
+			__unstableInitSource="core"
```

Comment:
What does "core" mean? I thought `__unstableInitSource` was just a prop for the menu item fill to be able to prioritize which menu item to show?

Reply:
We can always add a new prop that allows disabling auto-injecting the menu item by the sidebar. What do you think?